patches/0002-AppStream-fix-tags.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/data/com.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml b/data/com.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml
+index bdc490d..174cf57 100644
+--- a/data/com.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml
++++ b/data/com.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml
+@@ -3,6 +3,7 @@
+ 	<id>com.github.thetumultuousunicornofdarkness.cpu-x</id>
+ 	<metadata_license>CC0-1.0</metadata_license>
+ 	<project_license>GPL-3.0+</project_license>
++	<developer_name>The Tumultuous Unicorn Of Darkness</developer_name>
+ 	<name>CPU-X</name>
+ 	<summary>Gathers information on CPU, motherboard and more</summary>
+ 	<description>
+@@ -15,12 +16,23 @@
+ 	<url type="homepage">https://thetumultuousunicornofdarkness.github.io/CPU-X/</url>
+ 	<url type="bugtracker">https://github.com/TheTumultuousUnicornOfDarkness/CPU-X/issues</url>
+ 	<url type="translate">https://hosted.weblate.org/engage/cpu-x/</url>
++	<icon type="remote" height="128" width="128">https://raw.githubusercontent.com/TheTumultuousUnicornOfDarkness/CPU-X/master/data/icons/CPU-X_128x128.png</icon>
```

Comment:
Don't add a remote icon

Reply:
Ok, done in c943e6d8730d002b60ba5d2181bcc960be4bfdc9.